app.js

```diff
@@ -0,0 +1,37 @@
+let form = document.querySelector('form');
+
+form.addEventListener('submit', async function (event) {
+    event.preventDefault();
+    console.log(form.username.value);
+    let repoNames = await getGitHubRepo(form.username.value);
+    displayRepos(repoNames);
+})
+
+async function getGitHubRepo(username) {
```

Comment:
First of all, this returns a list of repos. So should be in plural.

Secondly, a good practice is to name these functions starting with the word `fetch`. For example here it would be: `fetchGithubRepos`